www/shared/styles/base/typography.css

```diff
@@ -24,22 +24,33 @@ body {
 h1,
 h2,
 h3,
-h4 {
-    margin: 0.7em 0;
+h4,
+h5,
+h6,
+p {
+    margin: 0;
 }
 
 h1 {
-    @mixin typography-h1 rem;
+    @mixin --type-h1;
```

Comment:
`type` is too generic of a name. `typography` gives better info of what kind of mixin we are using. Also, the file is called `typography` already :D

Reply:
agree!